.github/workflows/release.yaml

```
@@ -24,16 +24,18 @@ jobs:
           - {
               name: "Windows Server 2022 / Qt 6",
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 12 but found 14

(indentation)